src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockPageTabs.tsx

```diff
@@ -171,14 +171,18 @@ const DataBlockPageTabs = ({
         const majorAxis = draft[0]?.axes?.major;
         const legend = draft[0]?.legend;
 
-        // Remove data sets that are no longer applicable to a given table's subject meta.
+        // If old chart title is the same as the old table title, then the chart title is defaulting to
+        // the table title, so don't inadvertently set the old chart/table title
+        if (dataBlock?.charts[0]?.title === dataBlock?.heading) {
+          draft[0].title = undefined;
```

Comment:
I got an exception at this point while testing on this branch. Mind taking a look into it plz?  This is after trying to create a Chart for a new data block (for a pre-existing Subject).

```
Unhandled Rejection (TypeError): Cannot set property 'title' of undefined

 174 | // If old chart title is the same as the old table title, then the chart title is defaulting to
  175 | // the table title, so don't inadvertently set the old chart/table title
  176 | if (dataBlock?.charts[0]?.title === dataBlock?.heading) {
> 177 |   draft[0].title = undefined;
      | ^  178 | }
  179 | 
  180 | // Remove data sets that are no longer applicable to a given table's subject meta.
```